src/bootstrap.js

```diff
@@ -0,0 +1,34 @@
+import has, { add, exists } from '@dojo/framework/has/has';
```

Comment:
this module will need copying over as part of ejecting. and we need to make sure any sniffs are not bound to `@dojo/cli-build-app/bootstrap` as won't work on ejection.